Vagrantfile

```diff
@@ -0,0 +1,45 @@
+Vagrant.configure(2) do |config|
```

Comment:
I'm using rbenv and it worked fine :)

On Sun, Dec 20, 2015 at 12:05 AM, stereobooster <notifications@github.com>
wrote:

> In Vagrantfile
> <https://github.com/grosser/wwtd/pull/37#discussion_r48100831>:
>
> > @@ -0,0 +1,45 @@
> > +Vagrant.configure(2) do |config|
>
> It's not about installing them locally, its about reproducible testing
> environment. With this one file, everybody who want contribute can get it
> with ease (without need to install rvm to your own machine, I prefer rbenv
> and don't want to mess with it)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/wwtd/pull/37/files#r48100831>.
>